pkg/util/testing/wrapper.go

```diff
@@ -1018,8 +1018,13 @@ func (m *MLPolicySourceWrapper) TorchPolicy(numProcPerNode string, elasticPolicy
 	if m.Torch == nil {
 		m.Torch = &trainer.TorchMLPolicySource{}
 	}
+	// For no numProcPerNode UT scenario
+	var numProcPerNodePtr *intstr.IntOrString
+	if numProcPerNode != "" {
+		numProcPerNodePtr = ptr.To(intstr.FromString(numProcPerNode))
+	}
```

Comment:
Instead of these hacking, we could use `numProcPerNode` as a pointer like 

```
func (m *MLPolicySourceWrapper) TorchPolicy(numProcPerNode *intstr.IntOrString, elasticPolicy *trainer.TorchElasticPolicy) *MLPolicySourceWrapper {
```

Reply:
Thanks, I have removed it and replaced them like `TorchPolicy(ptr.To(intstr.FromString("auto")), nil)`.